<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionLayout</class>
 <widget class="QMainWindow" name="PositionLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Position Layout</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>112</width>
       <height>144</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="centerRadioButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="leftUpRadioButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Left up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="leftDownRadioButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Left down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rightUpRadioButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Right up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rightDownRadioButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Right down</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>141</width>
       <height>129</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="AxisLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>X-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="XAxisSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Y-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="YAxisSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sizeSpinBox">
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>296</y>
       <width>136</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(243, 243, 243);
border-color: rgb(238, 238, 236);</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>121</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="boldCheckBox">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="italicCheckBox">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="underlineCheckBox">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Underline</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="colorBotton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>270</y>
       <width>17</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Text color</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>417</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Shadow
Offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="shadowOffsetSpinBox">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Shadow
Blur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="shadowBlurSpinBox">
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>centerRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>center_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftUpRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>left_up_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightUpRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>right_up_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightDownRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>right_down_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftDownRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>left_down_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_font_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontComboBox</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_font_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boldCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_font_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>italicCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_font_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>underlineCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_font_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorBotton</sender>
   <signal>clicked()</signal>
   <receiver>PositionLayout</receiver>
   <slot>change_color_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XAxisSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_X_axis_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YAxisSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_Y_axis_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shadowBlurSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_shadow_blur_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shadowOffsetSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PositionLayout</receiver>
   <slot>update_shadow_offset_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>center_slot()</slot>
  <slot>left_up_slot()</slot>
  <slot>left_down_slot()</slot>
  <slot>right_up_slot()</slot>
  <slot>right_down_slot()</slot>
  <slot>update_X_axis_slot()</slot>
  <slot>update_font_slot()</slot>
  <slot>update_Y_axis_slot()</slot>
  <slot>change_color_slot()</slot>
  <slot>update_shadow_blur_slot()</slot>
  <slot>update_shadow_offset_slot()</slot>
 </slots>
</ui>
